src/Screens/ItemDetail/index.tsx

```diff
@@ -44,7 +45,8 @@ const ItemDetail = ({navigation,route}) =>{
                      <Text style={styles.detailPrice}>Rs. {item.item.price}</Text>
 
                      <Button name = {'Add To Cart'} onPress={()=>{
-                         navigation.navigate('cart',item)
+                         navigation.navigate('cart',item),
+                         addToCart(item.item.id)
```

Comment:
Whatever objects(items) are added into the cart add them into array and store that array into the async storage ... so that when app is reopened you can show that data